dask/dataframe/operation.py

```diff
@@ -2,7 +2,9 @@
 
 from dataclasses import dataclass, replace
 from functools import cached_property
-from typing import Any, Callable, Hashable
+from typing import Any, Callable
+
+from typing_extensions import TypeAlias
```

Comment:
Would require adding this to the requirements, but it's already added in `distributed`, so I wouldn't be too worried